src/MOI_wrapper.jl

```diff
@@ -98,45 +98,76 @@ end
 using Compat.SparseArrays
 
 const ZeroCones = Union{MOI.EqualTo, MOI.Zeros}
-const LPCones = Union{MOI.GreaterThan, MOI.LessThan, MOI.Nonnegatives, MOI.Nonpositives}
+const LPCones = Union{MOI.GreaterThan, MOI.LessThan, 
+                      MOI.Nonnegatives, MOI.Nonpositives}
 
 # Computes cone dimensions
-constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:ZeroCones}) = ci.value
-#_allocateconstraint: Allocate indices for the constraint `f`-in-`s` using information in `cone` and then update `cone`
+constroffset(cone::ConeData, ci::CI{<:MOI.AbstractFunction, <:ZeroCones}) =
```

Comment:
Do not break a single line function into multiple lines, do `function constroffset(...) ..` instead

Reply:
It should be written in http://www.juliaopt.org/JuMP.jl/dev/style/#TODO:-Line-breaks-1

Reply:
By the way, I did not see either the 80-character limit in the style guide.